plugin/skills/commit-push/examples.md

```
@@ -0,0 +1,142 @@
+git status
+git log --oneline --graph "origin/$(gh repo view --json defaultBranchRef -q .defaultBranchRef.name)..HEAD"
+# (no commits yet on this branch)
+
+# 2. 戦略判断: ブランチに初めてのgitコミット → 新規gitコミット
+git add -A
+git commit
+```
+
+gitコミットメッセージ例:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)